e2e-tests/cypress/tests/integration/channels/files_and_attachments/file_preview_image_spec.js

```diff
@@ -103,6 +103,18 @@ describe('Upload Files - Image', () => {
 
         testImage(properties);
     });
+
+    it('MM-T2264_7 - WEBP', () => {
+        const properties = {
+            filePath: '../../../WEBP.webp',
```

Comment:
@saturninoabril please give the access to the gDrive so i can upload the webp image there to be used here